notifications/http.go

```diff
@@ -83,6 +84,7 @@ func (hs *httpSink) Write(events ...Event) error {
 
 		return fmt.Errorf("%v: error posting: %v", hs, err)
 	}
+	defer resp.Body.Close()
```

Comment:
Ooh, that's a good one.  I presume we were leaving connections in TIME_WAIT without this?